### PR TITLE
Login Component - first commit

### PR DESCRIPTION
**Pull request by BorisLaboratory**
Please review and approve

### CODE REVIEW

File: front-end/src/components/aws-exports.js
Comment (dellydel) [inline front-end/src/components/aws-exports.js:1]: This file can be removed if no longer being used

File: front-end/src/App.css [hunk 1]
```diff
@@ -1,6 +1,18 @@
 .App {
   text-align: center;
 }
+/* .amplify-button[data-variation='primary'] {
```
Comment (dellydel) [inline front-end/src/App.css:4]: No need to include commented out code when checking in your code.

File: front-end/package.json [hunk 2]
```diff
@@ -3,15 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@aws-amplify/ui-react": "^5.1.1",
```
Comment (dellydel) [inline front-end/package.json:6]: The amplify dependencies can be uninstalled

**Review by dellydel — CHANGES_REQUESTED** [review]
A few changed to remove things that are no longer needed

File: front-end/src/components/aws-exports.js
Comment (BorisLaboratory) [reply front-end/src/components/aws-exports.js:1]: export.js deleted

File: front-end/src/App.css (hunk 1 above)
Comment (BorisLaboratory) [reply front-end/src/App.css]: comments removed from App.css

**Review by dellydel — APPROVED** [review]
Nice work, approved

**Pull request merged**